code/src/components/Map.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import '@lottiefiles/lottie-player';
+
+export const Map = ({ history, gameDetails }) => {
+	const mapArray = ['0,0', '0,1', '0,2', '0,3', '1,0', '1,1', '1,2', '1,3'];
```

Comment:
There is an issue with this approach.  If you don't know the entire coordinate system or if the maze is backend-generated, you might run into issues here